templates/default.j2

```diff
@@ -0,0 +1,6 @@
+ZOO_LOG_DIR={{ zookeeper_log_dir }}
+ZOO_LOG4J_PROP=INFO,ROLLINGFILE
+
+{% for key, value in zookeeper_environment.iteritems() %}
```

Comment:
Having looked at the below document it looks like ```iteritems()``` isn't supported by Python 3. ```dict.items()``` should be used instead to be compatible with both Python 2 and 3:

https://docs.ansible.com/ansible/2.5/user_guide/playbooks_python_version.html